app/demo-interactor/PhysicsArrayCalculator.hh

```diff
@@ -0,0 +1,50 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file PhysicsArrayCalculator.hh
```

Comment:
Is this class specifically for KNDemo or should be in physics/base?

Reply:
The idea was to make this class a starting point for a full physics vector/table-type class that would move into physics/base (or physics/em if it's only applicable to the EM models and the geant4 silly cross section scaling). So for expediency I've left it here in the celeritas namespace as a straw-man class. I think once we get a second model/process fully implemented we can start generalizing it. Sound reasonable to you?